ballerina/tests/execute-basic-test.bal

```diff
@@ -334,6 +329,7 @@ type ResultCount record {
 }
 function testUpdateData() returns error? {
     MockClient dbClient = check new (url = executeDb, user = user, password = password);
+    // Usage of string in the execute API is depreciated
     ExecutionResult result = check dbClient->execute("Update NumericTypes set int_type = 11 where int_type = 10");
```

Comment:
why we do not change to parameterized query here?

Reply:
we might remove the string option from all APIs soon. So better we move to parameterized queries

Reply:
I have left one usage of each API as it is to test the string usage for now. All the others were changed.

Reply:
Okay, let's remove them when we remove the support completely